src/screens/HyperSwitch/SDKPayment/SDKPage.res

```diff
@@ -0,0 +1,197 @@
+      }
+    })
+
+    let selectProfileField = FormRenderer.makeFieldInfo(
+      ~label="Business profile",
+      ~name="profile_id",
+      ~placeholder="",
+      ~customInput=InputFields.selectInput(
+        ~deselectDisable=true,
+        ~options={arrayOfBusinessProfile->businessProfileNameDropDownOption},
+        ~buttonText="Select Profile",
+        ~disableSelect=disableSelectionForProfile,
+        ~fullLength=true,
+        (),
+      ),
+      (),
+    )
+    let selectProfileId = FormRenderer.makeFieldInfo(
+      ~label="Profile Id",
+      ~name="profile_id",
+      ~placeholder="",
+      ~customInput=InputFields.selectInput(
+        ~deselectDisable=true,
+        ~options=arrayOfBusinessProfile->businessProfileIdDropDownOption,
+        ~buttonText="Select Profile Id",
+        ~disableSelect=disableSelectionForProfile,
+        ~fullLength=true,
+        (),
+      ),
+      (),
+    )
+    let selectCurrencyField = FormRenderer.makeFieldInfo(
+      ~label="Currency",
+      ~name="currency",
+      ~placeholder="",
+      ~customInput=InputFields.selectInput(
+        ~options=dropDownOptions,
+        ~buttonText="Select Currency",
+        ~deselectDisable=true,
+        ~fullLength=true,
+        (),
+      ),
+      (),
+    )
+    let enterAmountField = FormRenderer.makeFieldInfo(
+      ~label="Enter amount",
+      ~name="amount",
+      ~placeholder="Enter amount",
+      ~customInput=InputFields.textInput(~isDisabled=false, ~customStyle="w-full", ()),
+      (),
+    )
+
+    <>
+      <FormRenderer.FieldRenderer field=selectProfileField fieldWrapperClass="!w-full" />
+      <FormRenderer.FieldRenderer field=selectProfileId fieldWrapperClass="!w-full" />
+      <FormRenderer.FieldRenderer field=selectCurrencyField fieldWrapperClass="!w-full" />
+      <FormRenderer.FieldRenderer field=enterAmountField fieldWrapperClass="!w-full" />
+      <FormRenderer.SubmitButton text="Show preview" />
+    </>
+  }
+}
+
+@react.component
+let make = () => {
+  open HSwitchMerchantAccountUtils
+  let hyperswitchMixPanel = HSMixPanel.useSendEvent()
+  let url = RescriptReactRouter.useUrl()
+  let filtersFromUrl = url.search->LogicUtils.getDictFromUrlSearchParams
+  let (currency, setCurrency) = React.useState(() => "USD")
+  let (isSDKOpen, setIsSDKOpen) = React.useState(_ => false)
+  let (key, setKey) = React.useState(_ => "")
+  let businessProfiles = Recoil.useRecoilValueFromAtom(HyperswitchAtom.businessProfilesAtom)
+  let defaultBusinessProfile = businessProfiles->getValueFromBusinessProfile
+
+  let initialDict =
+    [
+      ("currency", "United States-USD"->Js.Json.string),
+      ("amount", "100"->Js.Json.string),
+      ("profile_id", defaultBusinessProfile.profile_id->Js.Json.string),
+    ]->Js.Dict.fromArray
+
+  let (profile, setProfile) = React.useState(_ => defaultBusinessProfile.profile_id)
+  let (amount, setAmount) = React.useState(() => 10000)
+
+  React.useEffect1(() => {
```

Comment:
Please check this useEffect1

Reply:
This use effect is needed to show the payment status after the payment has been done